frontend/workflows/projectSelector/src/card.tsx

```diff
@@ -6,6 +6,7 @@ import { Grid, LinearProgress } from "@material-ui/core";
 
 const StyledProgressContainer = styled.div({
   height: "4px",
+  background: "rgb(235, 237, 251)",
```

Comment:
i would style the CardHeader fragment with background instead so it doesn't have to match

Reply:
updated